flow/Makefile

```diff
@@ -485,13 +485,13 @@ clean_route:
 	rm -f  $(REPORTS_DIR)/5_*
 	rm -f  $(LOG_DIR)/5_*
 
-klayout_tr_rpt: $(RESULTS_DIR)/5_route.def $(OBJECTS_DIR)/klayout.lyt
+klayout_tr_rpt: $(RESULTS_DIR)/5_route.def $(REPORTS_DIR)/5_route_drc.rpt $(OBJECTS_DIR)/klayout.lyt
```

Comment:
I'm not sure this is the best idea because there is no rule to make `$(REPORTS_DIR)/5_route_drc.rpt`. It is a byproduct of creating `$(RESULTS_DIR)/5_route.def`.

Reply:
I added this since the 5_route.def file can be very large and take a while to read in. So if the _drc.rpt  file is missing, this just stops the command early. I'm not wed to this, so if you think it's best to leave these two klayout_tr_* without the extra dependency, I'm happy to remove them

Reply:
Well I haven't tested it, but I believe that if you make `klayout_tr_rpt` without having run routing, then because `make` statically analyzes the dependency graph it will return `Error: no rule to make $(REPORTS_DIR)/5_route_drc.rpt` instead of generating the dependency. So I think it's best to leave it out.

Reply:
I'll remove them (this and the next one). Ideally the OR gui would be able to display the DRC violations and then the need for this would effectively go away.